### ISSUE TITLE
Support for SecureCRT

### ISSUE THREAD

**Issue by txynidakis** [romkatv/powerlevel10k#2746]
Hi
Wondering if anyone has setup guides for SecureCRT (https://www.vandyke.com/products/securecrt/)

When running though `pk10 configure` there is not steps to verify the fonts so the command line icons don't appear.

**Comment by romkatv** [comment]
See https://github.com/romkatv/powerlevel10k?tab=readme-ov-file#some-prompt-styles-are-missing-from-the-configuration-wizard

**Comment by txynidakis** [comment]
Thanks, @romkatv I found the issue!

Thanks for the FAQ tip. I ran `print $terminfo[colours]`, and the result was 8.

I changed the 'Emulation' settings from 'Linux' to 'Xterm', and then I was able to select 256 colours or 'True Color,' which solved the issue.

For additional info for others.
* Manually installed the fonts
* Selected the font on Sesision Options -> Terminal -> Appearance -> Select the font `MesloLGS NF` 

Kind regards
